physics/docs/pdftxt/GFSv16beta_suite.txt

```diff
@@ -29,7 +29,7 @@ The GFS_v16beta physics suite uses the parameterizations in the following order:
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_GFS_v16beta" lib="ccppphys" ver="5">
```

Comment:
Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.